src/colmap/scene/reconstruction.cc

```diff
@@ -452,8 +452,8 @@ Reconstruction Reconstruction::Crop(
   for (const auto& image : images_) {
     auto new_image = image.second;
     new_image.SetRegistered(false);
-    for (auto& point2D : new_image.Points2D()) {
-      point2D.point3D_id = kInvalidPoint3DId;
+    for (point2D_t pid = 0; pid < new_image.NumPoints2D(); ++pid) {
+      new_image.ResetPoint3DForPoint2D(pid);
```

Comment:
```suggestion
      new_image.ResetPoint3DForPoint2D(p oint2D_idx);
```

Reply:
Changed with remove white space in https://github.com/colmap/colmap/pull/2557/commits/6f716b8cdc107e076313934578a8a3eeeeab5ee3